### DISCUSSION TITLE
When will this packages get the status final?

### DISCUSSION THREAD

**Discussion by rkops-bd** [mono/SkiaSharp.Extended#168] [General]
This package is now 'in preview'. But when will this be available as stable?

**Comment by pvoelker** [comment]
The last nuget package release I see for 'SkiaSharp.Extended.UI.Maui' was 2.0.0-preview.61.  Which was a while ago.  Is this project going inactive?  Or has this moved somewhere else?  The lottie support in MAUI is great for the project I am working on....
